blueprints/lib/files/lib/.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
I don't actually want a change here, we need extension points for blueprints that we don't have yet; I consider this a reasonable compromise.